conf/config.yaml

```diff
@@ -65,6 +65,11 @@ nginx_config:                     # config for render the template to genarate n
     client_header_timeout: 60s     # timeout for reading client request header, then 408 (Request Time-out) error is returned to the client
     client_body_timeout: 60s       # timeout for reading client request body, then 408 (Request Time-out) error is returned to the client
     send_timeout: 10s              # timeout for transmitting a response to the client.then the connection is closed
+  server:                          # set custusmized directives in nginx.conf server field  
+    client_body_buffer_size: 16k   # Sets buffer size for reading client request body
```

Comment:
I think you can comment these two lines.

Reply:
comment done